Dockerfile

```diff
@@ -2,15 +2,27 @@ FROM eclipse-temurin:11-jdk as build
 
 WORKDIR /tmp
```

Comment:
this line could be removed as well?

Reply:
No, the ./gradlew war command depends on it

Reply:
Fixed